casepro/statistics/signals.py

```diff
@@ -48,23 +48,27 @@ def record_new_outgoing(sender, instance, created, **kwargs):
                 seconds_since_open = ceil(td.total_seconds())
                 DailySecondTotalCount.record_item(day, seconds_since_open,
                                                   DailySecondTotalCount.TYPE_TILL_REPLIED, org)
-
+            # don't count self-assigned cases
```

Comment:
This comment being here is a bit misleading as the check for this is actually line 59